packages/dapp/src/components/DualWidget/DualWidget.style.tsx

```diff
@@ -1,19 +1,162 @@
 import { Grid, GridProps } from '@mui/material';
-import { Breakpoint, styled } from '@mui/material/styles';
+import { styled } from '@mui/material/styles';
 
 export interface WidgetContainerProps extends Omit<GridProps, 'component'> {
   isActive?: boolean;
+  showWelcome?: boolean;
+  showFadeOut?: boolean;
 }
 
 export const WidgetContainer = styled(Grid, {
-  shouldForwardProp: (prop) => prop !== 'isActive',
-})<WidgetContainerProps>(({ theme, isActive }) => ({
+  shouldForwardProp: (prop) =>
+    prop !== 'isActive' && prop !== 'showWelcome' && prop !== 'showFadeOut',
+})<WidgetContainerProps>(({ theme, isActive, showWelcome, showFadeOut }) => ({
   display: isActive ? 'inherit' : 'none',
-  paddingTop: theme.spacing(2),
-  [theme.breakpoints.up('sm' as Breakpoint)]: {
-    paddingTop: theme.spacing(2),
+  height: showWelcome && !showFadeOut ? '45vh' : '100%',
+  placeContent: 'center',
+  paddingTop: showWelcome ? 'unset' : theme.spacing(6),
+  minHeight: showWelcome && '250px',
+  zIndex: showWelcome && 1500,
+
+  [`@media screen and (max-height: 490px)`]: {
+    display: showWelcome && 'none',
+
+    '& > .onramper-container, & .widget-wrapper': {
+      top: showWelcome && '50%',
+    },
+  },
+
+  '&:after': {
+    content: showWelcome && !showFadeOut && '" "',
+    background:
+      showWelcome && theme.palette.mode === 'dark'
+        ? 'linear-gradient(to top, #200a47 0%, transparent 100%)'
+        : showWelcome && theme.palette.mode === 'light'
+        ? 'linear-gradient(to top, #e8dafb 0%, transparent 100%)'
+        : 'transparent',
+    position: 'absolute',
+    height: '150px',
+    pointerEvents: 'none',
+    left: 0,
+    right: 0,
+    bottom: '-4px',
+    transitionProperty: showWelcome && 'height, bottom',
+    transitionDuration: showWelcome && '.3s',
+    transitionTimingFunction: showWelcome && 'ease-in-out',
+  },
```

Comment:
This chunk replaces the maskImage which was used before. It gives a gradient effect as a transition between the bottom welcome part and the top part of the widget container

Reply:
It doesn't work well for a dark theme. You can see on the screenshot there is a gradient on both sides, and it looks like a faded lightsaber in the middle of the screen. It should only be to the upside.

<img width="1008" alt="image" src="https://github.com/lifinance/jumper.exchange/assets/18644653/150fcc03-ce27-41aa-a9c4-e9bbdc3205d8">

Reply:
Yes, that looked odd! Thank you for that idea, just fixed it. It is still just a color I picked and hard-coded here but to me it seems to adapt better to the rest of the background:

![image](https://github.com/lifinance/jumper.exchange/assets/43956540/ecd405c6-d1f3-4984-829e-927faff24be6)